### PR TITLE
Add instructions to set custom OS icons in rEFInd

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

File: docs/guides/refind.md [hunk 1]
```diff
@@ -149,6 +149,14 @@ By default, rEFInd boots Linux indirectly by booting GRUB, systemd-boot etc. But
 
 8. Now, when you shall be in rEFInd, it should show an entry with the path of the image of your kernel and shall most probably have the icon of the Linux Penguin. That entry shall be the one which shall boot Linux directly using rEFInd.
 
+9. If you want to use your distro's icon instead of Linux Penguin one, you may label the volume containing your kernel with the name of your distro. For example if the distro is **Ubuntu** and kernel is in `/dev/nvme0n1p3` partition, run:-
+
+    ```plain
+    sudo tune2fs -L Ubuntu /dev/nvme0n1p3
```
Comment (Redecorating) [inline docs/guides/refind.md:155]: I think this'll only work on ext2/3/4 filesystems, some people may be using btrfs, zfs, etc.

**Review by Redecorating — CHANGES_REQUESTED** [review]

File: docs/guides/refind.md (hunk 1 above)
Comment (AdityaGarg8) [reply docs/guides/refind.md:155]: That's why I added the link to rEFInd's official guide

Comment (AdityaGarg8) [reply docs/guides/refind.md:155]: `sudo btrfs filesystem label <mountpoint> <newlabel>`

Is this for btrfs?

Comment (AdityaGarg8) [reply docs/guides/refind.md:155]: I probably can add instructions to label common fs in use?

Comment (Redecorating) [reply docs/guides/refind.md:155]: > `sudo btrfs filesystem label <mountpoint> <newlabel>`
> 
> 
> 
> Is this for btrfs?

That looks right but I can't check right now.

Comment (AdityaGarg8) [reply docs/guides/refind.md]: Any other common FS @Redecorating?

Comment (AdityaGarg8) [reply docs/guides/refind.md]: I cannot format to zfs BTW for testing, nor do I know any command that does so. But I believe that its not used that much anyways.

Comment (appleplectic) [reply docs/guides/refind.md]: JFS perhaps?

Comment (AdityaGarg8) [reply docs/guides/refind.md]: Ubuntu won't let me format to that either. If you know how to set a label, share here.

**Review by Redecorating — APPROVED** [review]

**Pull request merged**